shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -681,146 +681,176 @@ - (void)testSetFlutterViewControllerInTheMiddleOfTouchEventShouldStillAllowGestu
       break;
     }
   }
-  UIViewController* mockFlutterViewContoller = OCMClassMock([UIViewController class]);
+  id mockFlutterViewContoller = OCMClassMock([UIViewController class]);
   {
     // ***** Sequence 1, finishing touch event with touchEnded ***** //
-
+    id event = OCMClassMock([UIEvent class]);
```

Comment:
Why are you sharing these events now?  I would have tried to limit the number of variables.

Reply:
Reverted, PTAL